app/recurring_job.go

```diff
@@ -326,6 +360,13 @@ func (job *Job) run() (err error) {
 		job.logger.Infof("Volume %v is in state %v", volumeName, volume.State)
 	}
 
+	// only recurring job types `snapshot` and `backup` need to check if old snapshots can be deleted or not before creating
+	if job.task == longhorn.RecurringJobTypeSnapshot || job.task == longhorn.RecurringJobTypeBackup {
```

Comment:
So the `recurring back` will be behavior change as well, and it will only take effect if there is no error found during the cleanup before doing the backup? 

My question is do we need `RecurringJobTypeBackupForceCreate`?

Reply:
Added another new recurring job type `backup-force-create`.
Thanks.